.gitignore

```diff
@@ -0,0 +1,155 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
Please make a separate directory for app, and another one for training notebooks

Reply:
Done